modules/custom/az_person/config/install/core.entity_form_display.node.az_person.default.yml

```diff
@@ -151,6 +152,14 @@ content:
     third_party_settings:
       media_library_edit:
         show_edit: '1'
+  field_az_netid:
+    type: string_textfield
+    weight: 6
+    region: content
+    settings:
+      size: 60
```

Comment:
For usability, field sizes should correspond with the size of the input expected from users. Since the max length of a NetID is 16 characters, the size could be set to 16 to match the expectation.

Reply:
> For usability, field sizes should correspond with the size of the input expected from users. Since the max length of a NetID is 16 characters, the size could be set to 16 to match the expectation.

Thanks @joshuasosa. We tested this in our 2024-07-24 PR meeting and found that using a size of 16 did not allow 16 characters to always fit (be visible) so we're going to change this to 30.